avaje-jex/src/main/java/io/avaje/jex/Context.java

```diff
@@ -361,7 +361,15 @@ default Context render(String name) {
    */
   String protocol();
 
-  class Cookie {
+  /**
+   * Gets basic-auth credentials from the request, or throws.
+   *
+   * <p>Returns a wrapper object containing the Base64 decoded username
+   * and password from the Authorization header, or null if basic-auth is not properly configured
+   */
+  BasicAuthCredentials basicAuthCredentials();
+
+  final class Cookie {
```

Comment:
I think Cookie should become an interface

Reply:
go for it, I'm working on the static file stuff